lib/manager/actions/versions.js

```diff
@@ -100,6 +100,12 @@ export function setActiveVersion (version: FeedVersion) {
     }
   }
 }
+// Sets the compared feed version.
+export const setComparedVersion = createAction(
```

Comment:
Maybe this could be renamed to `setComparedVersionId` and it could take an ID as the payload.

Reply:
With https://github.com/ibi-group/datatools-ui/pull/528/commits/03695eda1153b68d93751b1f0d27738edc2b41b1, I ended up placing the version object itself in the state and removed ids.